ValveResourceFormat/Resource/ResourceTypes/Texture.cs

```diff
@@ -87,6 +94,20 @@ public override void Read(BinaryReader reader, Resource resource)
 
                     reader.BaseStream.Position += offset;
 
+                    if (type == 3)
```

Comment:
Add this to `VTexExtraData` and reference it.

Reply:
Will do